src/Commands/SiteStudioColor.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Drupal\dst_entity_generate\Commands;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\dst_entity_generate\BaseEntityGenerate;
+use Drupal\dst_entity_generate\DstegConstants;
+
+/**
+ * Class provides functionality of Site Studio Integration for Color generation from DST sheet.
+ *
+ * @package Drupal\dst_entity_generate\Commands
+ */
+class SiteStudioColor extends BaseEntityGenerate {
+
+  /**
+   * {@inheritDoc}
+   */
+  protected $dstEntityName = 'site_studio_color';
+
+  /**
+   * Array of all dependent modules.
+   *
+   * @var array
+   */
+  protected $dependentModules = ['cohesion'];
+
+  /**
+   * Entity Type manager service.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * List of required fields to create entity.
+   *
+   * @var array
+   */
+  protected $requiredFields = ['id', 'machine_name'];
+
+  /**
+   * DstegMenu constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *   Entity Type manager.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager) {
+    $this->entityTypeManager = $entityTypeManager;
+  }
+
+  /**
+   * Generate all the Site Studio Colors from DEG sheet.
+   *
+   * @command deg:generate:sscolor
+   * @aliases deg:ssc
+   * @usage drush deg:ssc
+   *   Generates site studio colors if not present.
+   * @usage drush deg:ssc --update
+   *   Update the existing colors.
+   * @option update Update existing entity types and creates new if not present.
+   */
+  public function generateColor($options = ['update' => FALSE]) {
+    $this->io()->success('Generating Site Studio Color.');
+    $this->updateMode = $options['update'];
+    $mode = 'create';
+    if ($this->updateMode) {
+      $mode = 'update';
+    }
+    $data = $this->getDataFromSheet(DstegConstants::SITE_STUDIO_COLOR);
+    $cohesion_color_storage = \Drupal::entityTypeManager()->getStorage('cohesion_color');
```

Comment:
**Avoid directly calling Drupal::entityTypeManager(), instead inject the entity_type.manager service.**
Dependency injection is the preferred method for accessing and using services in Drupal 8 and should be used whenever possible.

 **References:**
 [Services and Dependency Injection](https://www.drupal.org/docs/8/api/services-and-dependency-injection/services-and-dependency-injection-in-drupal-8#using-dependency-injection)